pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -26,6 +26,7 @@ class DataConnector:
     def __init__(self, trainer: "pl.Trainer", multiple_trainloader_mode: str = "max_size_cycle"):
         self.trainer = trainer
         self.multiple_trainloader_mode = multiple_trainloader_mode
+        self.prefetcher: Optional[DataFetcher] = None
```

Comment:
how is this one considered if passed?

Reply:
Not sure to fully grasp the question.

Reply:
But we will address this when the `data_fetcher` is made public and can be passed through the Trainer arguments.